src/MCPServer/lib/jobChain.py

```diff
@@ -67,20 +64,11 @@ def __init__(self, unit, chainPK, notifyComplete=None, passVar=None, UUID=None,
         self.UUID = UUID
         self.linkSplitCount = 1
         self.subJobOf = subJobOf
-        sql = """SELECT * FROM MicroServiceChains WHERE pk =  '%s'""" % (chainPK.__str__())
-        print sql
-        c, sqlLock = databaseInterface.querySQL(sql)
-        row = c.fetchone()
-        if row == None:
-            sqlLock.release()
-            return None
-        while row != None:
-            print "jobChain", row
-            #self.pk = row[0]
-            self.startingChainLink = row[1]
-            self.description = row[2]
-            row = c.fetchone()
-        sqlLock.release()
+
+        chain = MicroServiceChain.objects.get(id=str(chainPK))
```

Comment:
The original code handles the case where there is no MicroServiceChain by returning None - this probably should as well.